src/main/java/gregtech/api/fluids/GTFluidRegistration.java

```diff
@@ -77,12 +77,14 @@ public void registerSprites(@NotNull TextureMap textureMap) {
      * @param generateBucket if a universal bucket entry should be generated
      */
     public void registerFluid(@NotNull Fluid fluid, @NotNull String modid, boolean generateBucket) {
-        fluidSprites.add(fluid.getStill());
-        fluidSprites.add(fluid.getFlowing());
-
         FluidRegistry.registerFluid(fluid);
-        fixFluidRegistryName(fluid, modid);
-
+        if (fluid instanceof GTFluid) {
```

Comment:
Can this use the return value of `FluidRegistry.registerFluid()`, or does this have to use instanceof to determine if it was our own fluid?